src/App.js

```diff
@@ -145,7 +177,23 @@ export default class App extends Component {
               loading={loading} />
           </SplitPane>
         </Content>
+
+        {errors.length > 0 ? (
+          <Notifications>
+            {errors.map(err => <Notification message={err} type='error' />)}
+          </Notifications>
+        ) : null}
       </Wrap>
     );
   }
 }
+
+function normalizeError(err) {
+  if (typeof err === 'object' && err.hasOwnProperty('message')) {
```

Comment:
Since we have a `service/api` to fetch the uast for a given code, I'd process the error there in order to return it in a expected way for the App instead of returning a raw server response, and handle here the error